swagger/v1/component_schemas/NPQParticipantProfileAttributes.yml

```diff
@@ -0,0 +1,27 @@
+description: "The data attributes associated with an NPQ application"
+type: object
+required:
+  - participant_id
+  - training_status
+properties:
+  id:
+    type: string
+    example: 7a8fef46-3c43-42c0-b3d5-1ba5904ba562
+    format: uuid
```

Comment:
This is not in the attributes, let's cut it.

Reply:
What attributes are you referring to?